v2/samples/datafactory/v1api/v1api20180601_factory.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: datafactory.azure.com/v1api20210601
+kind: Factory
+metadata:
+  name: samplefactory
+  namespace: default
+spec:
```

Comment:
Maybe add some more detail to the sample?

Reply:
I don't see other meaningful fields in data factory spec, added one more field - publicNetworkAccess